src/common/common.js

```diff
@@ -209,9 +215,10 @@ const Localizer = (function () {
             if (elem.hasAttribute(currentLocaleAttribute)) {
                 const attributeTag = elem.getAttribute(currentLocaleAttribute);
                 const translatedMessage = browser.i18n.getMessage(getMessageTag(attributeTag));
+                const isHTML = translatedMessage.startsWith("!HTML!");
                 // only set message if it could be retrieved, i.e. do not override HTML fallback
                 if (translatedMessage !== "") {
-                    elem.setAttribute(currentAttribute, translatedMessage);
+                    elem.setAttribute(currentAttribute, isHTML ? translatedMessage.replace(/!HTML!/, "").trim() : translatedMessage);
```

Comment:
Hmm, this assumes placeholders and stuff can never contain HTML… and yeah… it's true. So okay.
However, if someone uses !HTML! there the translation string is either invalid or that is actually wanted, so there is no real need to replace it here too, but I am fine with doing it anyway – it does not hurt.

But also here: leftTrim only, possible.